CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 
 ### Fixed
 * Fixed updating helpers (the `ClassMap`) used by `Realm` before notifying schema change listeners when the schema is changed during runtime. ([#5574](https://github.com/realm/realm-js/issues/5574))
+* Fix crashes on refresh of the React Native application. ([#5904](https://github.com/realm/realm-js/issues/5904))
```

Comment:
```suggestion
* Fix crashes on refresh of the React Native application. ([#5904](https://github.com/realm/realm-js/issues/5904), since v11.7.0)
```

Reply:
At least it is my guess that we introduced the regression in v11.7.0 (Realm Core v13.7.0 introduced the new logger)

Reply:
I'm not so sure.  We didn't actually implement a default logger in v11.  I can test this though.